importers/degree-importer.php

```diff
@@ -688,6 +692,12 @@ private function process_post_terms( $post_id, $post_terms ) {
 				}
 
 				if ( $term_id ) {
+					// Set the alias
+					$alias = $this->get_college_alias( $term );
+					if ( ! add_term_meta( $term_id, 'colleges_alias', $alias, true ) ) {
+						update_term_meta( $term_id, 'colleges_alias', $alias, true );
```

Comment:
update_term_meta() should add the meta field if it doesn't exist yet (https://developer.wordpress.org/reference/functions/update_term_meta/#description)--is there still a need for the add_term_meta() check in this case, or is it there just for clarity?